js/nina-common/src/utils/client.js

```diff
@@ -51,7 +51,7 @@ const ENDPOINT_PRESSING_PLANT =
     : 'https://pressingplant-dev.nina.market:443'
 const ENDPOINT_API =
   process.env.REACT_APP_CLUSTER === 'mainnet'
-    ? 'https://api.nina.market:443'
+    ? 'https://api-dev.nina.market:443'//'https://api.nina.market:443'
```

Comment:
is this intentional?

Reply:
this is because we are testing on the dev server right now - this would be changed to production when the corresponding code is live on produciton